lld/COFF/Writer.cpp

```diff
@@ -2584,6 +2586,16 @@ void Writer::createDynamicRelocs() {
                          coffHeaderOffset + offsetof(coff_file_header, Machine),
                          AMD64);
 
+  if (ctx.symtab.entry || ctx.hybridSymtab->entry) {
```

Comment:
Technically, a different way of expressing this condition would be `if (ctx.symtab.entry != ctx.hybridSymtab->entry)`, right? (Not a strong opinion though.)

Reply:
Yeah, it expresses the actual reason for the reloc, I will change it.